--- performous-1.2.0/game/ffmpeg.cc.orig	2022-04-18 23:50:35.461091985 +0100
+++ performous-1.2.0/game/ffmpeg.cc	2022-04-18 23:51:44.772857009 +0100
@@ -251,7 +251,7 @@ FFmpeg::FFmpeg(fs::path const& _filename
 	if (err < 0) throw Error(*this, err);
 	m_formatContext->flags |= AVFMT_FLAG_GENPTS;
 	// Find a track and open the codec
-	AVCodec* codec = nullptr;
+	const AVCodec* codec = nullptr;
 	m_streamId = av_find_best_stream(m_formatContext.get(), static_cast<AVMediaType>(mediaType), -1, -1, &codec, 0);
 	if (m_streamId < 0) throw Error(*this, m_streamId);
 
